parm/parm_wave/grib2_gfswave.ak_10m.f003

```diff
@@ -0,0 +1,16 @@
+&GRIBIDS DESC=' WIND     Surface ',WMOHEAD='EQEB88 KWBJ',PDTN= 0 ,PDT=  2 1 2 0 11 0 0 1 3 1 0 1 255 0 0 /
```

Comment:
It seems like a lot to have a separate part file for every forecast hour.  Is there a way to have a generic one?

Reply:
I don't know if that is possible, for all waves products awips (WMO) headers are provided by other group in EMC (Boi Vuong).

Reply:
So based on a quick look, for a particular grid, it seems like what is different is the forecast hour in each line of the file.  If we did a search and replace the forecast hour, then we could just have a header file for each grid instead of each forecast hour and cut down on the over all number of files.  This would require a scripting change, but I'm assuming it wouldn't be too big of one?

Reply:
Agree with Jessica. There has to be a better solution. It is going to be impossible to maintain like this.

Reply:
I will contact Boi about this.
awips atm products must be using the data structure for the wmo headers.

Reply:
I'm not saying to change the data structure, I'm just saying it looks like it could be templated to reduce the number of files here.  Hopefully Boi can provide more insight.

Reply:
This is how the other AWIPS cards exist (one for each forecast hour) so we can ingest these new files from Roberto as-is. Let's discuss templating them for the next model version. Thanks!